### PR TITLE
upgrade to parity 2.2.7

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM parity/parity:v1.11.1
+FROM parity/parity:v2.2.7
```
Comment (MicahZoltu) [inline Dockerfile:1]: That is a lot of new numbers... we are so far behind.

File: chain-spec.json [hunk 2]
```diff
@@ -14,7 +14,6 @@
 		"eip161dTransition": "0x0",
 		"eip155Transition": "0x0",
 		"eip98Transition": "0x7fffffffffffff",
-		"eip86Transition": "0x7fffffffffffff",
```
Comment (MicahZoltu) [inline chain-spec.json:17]: Why has EIP 86 been removed?

**Review by MicahZoltu — APPROVED** [review]

File: chain-spec.json (hunk 2 above)
Comment (epheph) [reply chain-spec.json:17]: https://github.com/paritytech/parity-ethereum/pull/9268

Comment (MicahZoltu) [reply chain-spec.json:17]: TL;DR: will take your word for it that it is a reasonable change.

**Pull request merged**